gradle.properties

```diff
@@ -12,7 +12,7 @@ syncAndroidModules=true
 # Version is not a day of a month. We don't want to bound releases to days.
 projectVersion=2020.2.6
 # Previous stable version to be used in this project
-infraVersion=2020.2.1
+infraVersion=2020.2.6
```

Comment:
Where will we use this version after this PR?

Reply:
it was released, but never integrated (process still not stable)

Reply:
i discovered a bug with incorrect plugin declaration because didn't integrate test version correctly, now we need .7 version